src/components/Button/index.js

```diff
@@ -34,13 +41,15 @@ Button.propTypes = {
   children: React.PropTypes.node,
   className: PropTypes.string,
   disabled: PropTypes.bool,
+  emphasized: PropTypes.bool,
```

Comment:
one thing I realised with `emphasized` is that we only have two options because it's boolean. Though imagine the style guide may want small medium big. It's not a big deal currently but worth checking with taahir.